modules/tensor_mechanics/src/kernels/StressDivergenceTensors.C

```diff
@@ -29,6 +31,7 @@ InputParameters validParams<StressDivergenceTensors>()
 StressDivergenceTensors::StressDivergenceTensors(const InputParameters & parameters) :
     Kernel(parameters),
     _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : ""),
+    _stress_old(getMaterialPropertyOldByName<RankTwoTensor>(_base_name + "stress")),
```

Comment:
No. Please do not just add stateful materials to our main tensor mechanics kernel! These come at a cost! Please conditionally request staeful properties if they are _really_ needed.